modules/caching-materials-manager-browser/src/index.ts

```diff
@@ -14,3 +14,4 @@
  */
 
 export * from './caching_materials_manager_browser'
+export { getLocalCryptographicMaterialsCache } from '@aws-crypto/cache-material'
```

Comment:
Where are these exports being used? Should they be included in a PR containing changes which include them?

Reply:
You can not create a local cache easily without these.  The should always have been exported...  I found this "bug" in writing the examples.  I can put it somewhere else, but I felt like they were in the same world.